priv/templates/slime/phx.gen.html/edit.html.eex

```diff
@@ -0,0 +1,8 @@
+section#torch-toolbar
```

Comment:
As noted above, can these files (here and other view templates below) just be `priv/templates/slime/phx.gen.html/edit.html.slime`?

Reply:
I see, now that I look at the phoenix-slime source code.  They actually name their view templates with an `.html.eex` extension which seems odd to me, but I assume you did it this way for consistency and/or because the generator failed completely if they weren't named this way?